DogeNetWebClient/src/app/store/reducers/index.ts

```diff
@@ -0,0 +1,27 @@
+import { ActionReducer, ActionReducerMap, MetaReducer } from '@ngrx/store';
+import { environment } from 'src/environments/environment';
+import { ConversationsState } from '../states/ConversationsState';
+import { MessagesState } from '../states/MessagesState';
+import { UserState } from '../states/UserState';
+import {
+  conversationsNode,
+  conversationsReducer,
+} from './conversations.reducer';
+import { messagesNode, messagesReducer } from './messages.reducer';
+import { userNode, userReducer } from './user-info.reducer';
+
+export interface State {
+  [userNode]: UserState;
+  [conversationsNode]: ConversationsState;
+  [messagesNode]: MessagesState;
+}
+
+export const reducers: ActionReducerMap<State, any> = {
+  [userNode]: userReducer,
+  [conversationsNode]: conversationsReducer,
+  [messagesNode]: messagesReducer,
+};
+
+export const metaReducers: MetaReducer<State, any>[] = !environment.production
```

Comment:
I don't see any logic here. Why do we have to check the environment and return the same empty array? Seems these meta reducers should be removed from the code.

Reply:
ok